save-core/src/jvmMain/kotlin/org/cqfn/save/core/utils/ProcessBuilderInternal.kt

```diff
@@ -47,7 +46,7 @@ actual class ProcessBuilderInternal actual constructor(
                 }
             )
         )
-        val data = br.lines().collect(Collectors.joining("\n"))
+        val data = br.lines().toList().joinToString("\n")
```

Comment:
```suggestion
        val data = br.lineSequence().joinToString("\n")
```
To avoid creation of an additional list; `lineSequence` is a kotlin extension instead of Java streams

Reply:
Done